docs/04-operation-guides/troubleshooting/apix-09-upgrade-sidecar-proxy.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Pods stuck in Pending/Failed/Unknown state after upgrade
+---
+
+## Symptom
+
+You cannot access services or functions using the API Rules created. The Kyma Gateway refuses the connection.
+Some of your Pods are stuck in the Pending/Failed/Unknown state, or their sidecar proxy version differ from the installed Istio version.
+
+## Cause
+
+During the upgrade Kyma will trigger rollout restart to instance of resources to ensure full compatibility of sidecar proxy with newly installed Istio version. Exception to this are standalone pods (without OwnerReference) or pods created by Job controller. Pods that cannot complete rollout restart process or rollout restart cannot be triggered because a pod is standalone or created by Job resource will get annotation `istio.reconciler.kyma-project.io/proxy-reset-warning` with brief explanation of the cause.
+
+## Remedy
+
+There are multiple reasons why Pods cannot become available and each case should be troubleshot separately. After resolving the root cause it's safe to perform rollout restart manually and remove annotation with proxy reset warning.
+
+In case of standalone pods or pods created by Job controller owner should recreate pods with incompatible sidecar proxy and verify if version match installed version (see commands: `istioctl version` and `istioctl proxy-status`)
```

Comment:
```suggestion
In case of standalone Pods or Pods created by the Job controller the owner should recreate Pods with incompatible sidecar proxy and verify if the Istio version match the installed version (see commands: `istioctl version` and `istioctl proxy-status`)
```
We are talking about the Istio version, right?